dao/src/main/java/com/garret/movies/dao/entity/Movie.java

```diff
@@ -0,0 +1,63 @@
+package com.garret.movies.dao.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.garret.movies.dao.util.MovieUtil;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name = "movie")
+public class Movie {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Date released;
+    private Double imdbRating;
+    private Integer imdbVotes;
+    @JsonProperty("Title")
+    private String title;
+    @JsonProperty("Runtime")
+    private String runtime;
+    @JsonProperty("Director")
+    private String director;
+    @JsonProperty("Plot")
+    @Column(length = 1024)
+    private String plot;
+    @JsonProperty("Type")
+    private String type;
+    @JsonProperty("imdbID")
+    @Column(nullable = false)
+    private String imdbId;
+    @OneToMany(cascade = CascadeType.ALL)
+    private List<Genre> genres;
+    @OneToMany(cascade = CascadeType.ALL)
+    private List<Actor> actors;
+    @OneToMany(cascade = CascadeType.ALL)
+    private List<Language> languages;
+    @OneToMany(cascade = CascadeType.ALL)
+    private List<Country> countries;
+
+    public Movie(@JsonProperty("Released") String released,
```

Comment:
@Garret-vin Why DB model has Json creator config?

Reply:
> @Garret-vin Why DB model has Json creator config?

@mmarusich Because types of fields in the model are different from json-response. In creator i convert its via MovieUtil class.